usaspending_api/disaster/v2/views/federal_account/federal_account_result.py

```diff
@@ -0,0 +1,67 @@
+from dataclasses import dataclass, field
+from dataclasses_json import dataclass_json
+from typing import Dict
+
+from usaspending_api.common.data_classes import Pagination
+from usaspending_api.disaster.v2.views.data_classes import Collation, Element
+
+
+class TAS(Element):
+    """Renaming the original generic object to clearly be TAS"""
+
+
+class FedAccount(Collation):
+    """Renaming the original generic object to clearly be Federal Account"""
+
+
+@dataclass_json
+@dataclass
+class FedAcctResults:
+    _federal_accounts: Dict[FedAccount, FedAccount] = field(default_factory=dict)
```

Comment:
Should this be a `field(default_factory=list)` since the Key and Value are the exact same thing?

Reply:
It's still a dictionary.